frame/support/procedural/src/pallet/expand/storage.rs

```diff
@@ -86,12 +88,26 @@ fn check_prefix_duplicates(
 	Ok(())
 }
 
+pub struct OnEmptyStructMetadata {
```

Comment:
The struct name seems to indicate that this is a general purpose struct. However, it is only usable for `ResultQuery` `OnEmpty`?